### PR TITLE
Quality/91 variable cleanup

### PR DESCRIPTION
**Pull request by danhahn**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [ ] One core team UX approval

### CODE REVIEW

**Comment by jondlm** [comment]
While you're at it, you should move the component specific variables into their own files. Specifically referring to:

```
// banner sizes
@Banner-size-spacing-rightLeft: 10px;
@Banner-size-spacing-topBottom: 13px;
@Banner-size-borderRadiaus: @borderRadius;

// button sizes
@Button-size-padding: 12px;
@Button-size-lineHeight: 26px;
@Button-size-borderRadius: @size-borderRadius;
```

File: src/components/Icon/EligibilityIcon/EligibilityIcon.less [hunk 1]
```diff
@@ -1,33 +1,33 @@
 .lucid-EligibilityIcon{
 	&-is-selected-neither{
 		.Icon-left, .Icon-right {
-			fill: @color-default;
-			stroke: @color-danger
+			fill: @featured-color-default;
+			stroke: @featured-color-danger
```
Comment (jondlm) [inline src/components/Icon/EligibilityIcon/EligibilityIcon.less]: Missing semi colons on these? Or is that something weird with stroke?

File: src/styles/mixins.less [hunk 2]
```diff
@@ -90,3 +90,10 @@
 	-webkit-transform:translate3d(0,0,0);
 }
 
+.dropshaow-gradient(@start, @stop, @height) {
```
Comment (jondlm) [inline src/styles/mixins.less]: typo

File: src/styles/variables.less [hunk 3]
```diff
@@ -58,42 +59,42 @@
...
+@featured-color-default-gradientEndColor: #e2e2e2;
 
 // primary
-@color-primary-borderColor: @color-primary;
-@color-primary-borderColorLite: fade(@color-primary, 40%);
-@color-primary-backgroundColor: fade(@color-primary, 30%);
-@color-primary-gradientStartColor: fade(@color-primary, 70%);
-@color-primary-gradientEndColor: @color-primary;
+@featured-color-primary-borderColor: @featured-color-primary;
+@featured-color-primary-borderColorLite: tint(@featured-color-primary, 60%);
```
Comment (jondlm) [inline src/styles/variables.less]: Nice use of tint! That should be our default unless we want the alpha that `fade` brings.

File: src/components/Icon/EligibilityIcon/EligibilityIcon.less (hunk 1 above)
Comment (danhahn) [inline src/components/Icon/EligibilityIcon/EligibilityIcon.less]: good catch

File: src/styles/mixins.less (hunk 2 above)
Comment (danhahn) [inline src/styles/mixins.less]: fixed

File: src/styles/variables.less (hunk 3 above)
Comment (danhahn) [inline src/styles/variables.less]: agreed

**Comment by nathaniellee** [comment]
Why was this ticket labeled with `semver-none`? Shouldn't it have been `semver-patch`? Just curious...

**Comment by danhahn** [comment]
@nathaniellee I just put the wrong one on

**Pull request merged**